browser/src/DatasetSelector.tsx

```diff
@@ -378,7 +385,7 @@ class NavigationMenu extends Component<Props, State> {
                   </TopLevelNavigationLink>
                   {/* @ts-expect-error TS(2769) FIXME: No overload matches this call. */}
                   <SubNavigationMenu isExpanded={isExpanded}>
-                    {(item as any).children.map((childItem: any) => (
+                    {(item).children.map((childItem) => (
```

Comment:
Nice, nickel and dime-ing our way through getting rid of the `any`s introduced with `ts-migrate` by way of stricter more specific types is always a good thing.